website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+### Input
+
+There are no specific inputs to the GCBM module. It contains several modules which have multiple inputs and they all interact with each other to produce the output of GCBM in the form of maps and tables.
+
+### Output
+
+GCBM gives us the estimate of carbon flux generated, Aboveground Biomass etc.
+
+|![Diagram depicting how the GCBM output is displayed on a chart.](../static/img/gcbm-chart-module-case-study.png) |
+|:--:|
+| Diagram depicting how the GCBM output is displayed on a chart. |
+
+|![Diagram depicting how the GCBM output is displayed on a map.](../static/img/gcbm-map-module-case-study.png) |
+|:--:|
+| Diagram depicting how the GCBM output is displayed on a map. |
+
+There is a special repository to visualize the output generated by GCBM [Taswira](https://github.com/moja-global/GCBM.Visualisation_Tool).
+
+## Frequently asked questions
+
+1. Do Modules need a specific version to run? 
+
+    The modules work with the [`develop`](https://github.com/moja-global/FLINT/tree/develop) branch of the **FLINT**. The actively maintained modules work with the most recent version (and are likely broken with old enough builds of the FLINT). The API has evolved a bit over time, so some modules might have fallen out of date but nothing requires a custom fork of FLINT.
+
+2. Why are Modules important?
+
+    Without the modules, the FLINT cannot produce outputs. Each user will have different data and policy and reporting needs. Based on these needs users will attach the appropriate modules and data to the FLINT. 
+
+3. Where can I learn more about modules?
+
+    For getting a deep understanding of modules, you have to understand how things work inside a module and what are the different operations used in FLINT and how it is being managed and developed. The [Metric and carbon calculation modules docs](https://github.com/moja-global/FLINT.Modules.docs/blob/develop/modules.md) offer an advance level of understanding.
+
+4. Can I implement a module from scratch?
+
+    Here's an example of a custom made module - [**Agricultural soil module**](https://github.com/moja-global/FLINT.Module.Agricultural_Soil_Model). This project involves specifically implementing the agricultural soils model that is used to estimate non-CO2 GHG emissions from agricultural practices. You can visit the [module development docs](https://github.com/moja-global/FLINT.Modules.docs/blob/develop/modules.md#developing-modules) to know how to implement a single, simple module from scratch.
+
+5. How do modules interact with each other?
+
+    Different Modules are configured through Local Domain Controller and sequencer which decides the flow of modules. Here is a **Sequence diagram** for better understanding.
+
+    |![Diagram depicting the sequence diagram of how FLINT modules interact with each other.](../static/img/module-interaction-module-case-study.png) |
+    |:--:|
+    | Diagram depicting the sequence diagram of how FLINT modules interact with each other. |
```

Comment:
I think this is probably not necessary for this case study. The control flow diagram is good though.